sbin/celeborn-ratis-shell-config.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# resolve links - $0 may be a softlink
+this="${BASH_SOURCE-$0}"
+common_bin=$(cd -P -- "$(dirname -- "${this}")" && pwd -P)
+script="$(basename -- "${this}")"
+this="${common_bin}/${script}"
+
+# convert relative path to absolute path
+config_bin=$(dirname "${this}")
+script=$(basename "${this}")
+config_bin=$(cd "${config_bin}"; pwd)
+this="${config_bin}/${script}"
+
+if [[ -e "${CELEBORN_CONF_DIR}/celeborn-ratis-shell-env.sh" ]]; then
+  . "${CELEBORN_CONF_DIR}/celeborn-ratis-shell-env.sh"
+fi
+
+# Check if java is found
+if [[ -z "${JAVA}" ]]; then
+  if [[ -n "${JAVA_HOME}" ]] && [[ -x "${JAVA_HOME}/bin/java" ]];  then
+    JAVA="${JAVA_HOME}/bin/java"
+  elif [[ -n "$(which java 2>/dev/null)" ]]; then
+    JAVA=$(which java)
+  else
+    echo "Error: Cannot find 'java' on path or under \$JAVA_HOME/bin/. Please set JAVA_HOME in ratis-shell-env.sh or user bash profile."
```

Comment:
"... in ratis-shell-env.sh" => "... in celeborn-ratis-shell-env.sh"

Reply:
> Cannot find 'java' on path or under \$JAVA_HOME/bin/

Strictly speaking, better to switch the order of $JAVA_HOME/bin/ and $PATH, because we search the $JAVA_HOME/bin/ first.

Reply:
Use the same code of celeborn-class, how about current?